repose-aggregator/components/services/http-client-service/http-client-service-api/src/main/resources/META-INF/schema/config/http-connection-pool.xsd

```diff
@@ -239,14 +239,22 @@
             </xs:annotation>
         </xs:attribute>
 
-        <xs:attribute name="chunked-encoding" type="xs:boolean" use="optional" default="true">
+        <xs:attribute name="chunked-encoding" type="chunkedEncodingType" use="optional" default="true">
             <xs:annotation>
                 <xs:documentation>
                     <html:p>
                         Determines whether or not this client should send chunked data when forwarding requests.
                     </html:p>
                     <html:p>
-                        WARNING: Setting this attribute will cause repose to attempt to retrieve the
+                        If set to true or 1, chunked data will always be sent.
+                        If set to false or 0, chunked data will never be sent.
+                        If set to auto, chunked data will only be sent if the original request to Repose was chunked.
+                        Note that the auto setting only has meaning for the default connection pool, and only when
+                        requests are forwarded to the origin service. A value of auto for non-default pools will
+                        behave identically to a value of true.
```

Comment:
It seems that we don't use chunked encoding for third party APIs. So I'll update this.